python/tests/graph/test_pieces.py

```diff
@@ -49,6 +49,8 @@
 
 
 class GraphPiecesTest(SparkDLTestCase):
+    
+    featurizerCompareDigitsExact = 5
```

Comment:
I was thinking we could put this all the way in `SparkDLTestCase` and use it everywhere we do the array almost comparison. But if this is blocking and we want to put that off for later, fine by me.

Reply:
It might not be a good idea to put the featurizerCompareDigitsExact in SparkDLTestCase. Because of the different condition numbers, we might need different number for different model. Like for  InceptionV3 and Xception, we make it to 4. Since `np.testing.test_arrays_almost_equal` already have a default value 6. We may not want to define a different one, unless we define the new  `assertArrayAlmostEqual ` as you suggested.

Reply:
Aha! did not know that. that is definitely more complicated.